backend/papersfeed/utils/recommendations/utils.py

```diff
@@ -83,14 +85,59 @@ def select_recommendation(args):
 
     recommendation_queryset = UserRecommendation.objects.filter(user_id=request_user.id)
 
-    recommendations = get_results_from_queryset(recommendation_queryset, 10, page_number)
+    recommendations = get_results_from_queryset(recommendation_queryset, 20, page_number)
 
     is_finished = not recommendations.has_next()
 
     recommendations = __pack_recommendations(recommendations, request_user)
 
     return recommendations, page_number, is_finished
 
+def select_keyword_init(args):
+    """Get keywords init"""
+
+    page_number = 1 if constants.PAGE_NUMBER not in args else int(args[constants.PAGE_NUMBER])
+
+    keyword_queryset = (Keyword.objects.annotate(num_papers=Count('papers'))
+                        .filter(num_papers__gt=9, num_papers__lt=100))
```

Comment:
I think this part is different from your explanation(keywords having 10 to 49 related papers). What is right?

Reply:
And I wonder why 'the number of keywords could be less than 20'. Could you explain? It filters by the number of related papers before pagination.

Reply:
That could happen when all keywords have just one related paper. I will change the explanation, Thank you

Reply:
So, you mean, it's due to `set`?

Reply:
No it is because of the filter